### ISSUE TITLE
Symbols

### ISSUE THREAD

**Issue by arybrm** [sspanak/tt9#693]
Thank you for such a good keyboard.
I wanted to ask about the symbols and smileys.
When I write without prediction and I want to get to the symbols, I press the number 1, and depending on how many times I press it, it moves.
But when I am in prediction mode, if I press the 1 key again, it will become a smiley,
And I tried to change the order in the settings, but it only changes when it is not in prediction.
So if it is possible to change this, even when it is prediction, it will work as usual.

And maybe if it is possible, then I wanted to ask more if it is possible to make such a screen of symbols and smileys, like there are on many phones, for example, where a long press on 1 or 0 will display a screen with many symbols that can be selected with the navigation keys.
I will support you again for this.

Thank you very much

**Comment by arybrm** [comment]
THANK YOU
Is it possible for a fee to arrange it so that it will be possible to choose it in a personalized way, also if it would be possible to arrange for a fee that there will also be speech to text because my phone does not have Google services so maybe really like someone once suggested that it will be possible for it to work with FUTO KEYBOARD.
I don't need an updated app, it doesn't need to be updated with all the changes you are going to introduce.
How can I contact you privately regarding payment etc.

**Comment by sspanak** [comment]
> Is it possible for a fee to arrange it so that it will be possible to choose it in a personalized way

No, sorry. This project is taking enough of my time even now. I don't have the capacity to work on personal requests. In fact, I don't think I'll be adding new features anymore.

> also if it would be possible to arrange for a fee that there will also be speech to text because my phone does not have Google services so maybe really like someone once suggested that it will be possible for it to work with FUTO KEYBOARD.

This is technically impossible. The Futo developers must implement the SpeechRecognizer API. Without it TT9 and Futo cannot communicate with each other. Also, Futo does not support all the languages TT9 does, so there is no guarantee it will solve the problem for people like you, who don't have Google. In short, it is not a matter of money, but of technology issues.
